fs/fs.go

```diff
@@ -559,78 +562,91 @@ func (self *RealFsInfo) GetDirFsDevice(dir string) (*DeviceInfo, error) {
 	return nil, fmt.Errorf("could not find device with major: %d, minor: %d in cached partitions map", major, minor)
 }
 
-func (self *RealFsInfo) GetDirDiskUsage(dir string, timeout time.Duration) (uint64, error) {
-	claimToken()
-	defer releaseToken()
-	return GetDirDiskUsage(dir, timeout)
-}
-
+// Deprecated. Use GetDirUsage instead.
 func GetDirDiskUsage(dir string, timeout time.Duration) (uint64, error) {
-	if dir == "" {
-		return 0, fmt.Errorf("invalid directory")
-	}
-	cmd := exec.Command("ionice", "-c3", "nice", "-n", "19", "du", "-s", dir)
-	stdoutp, err := cmd.StdoutPipe()
+	usage, err := GetDirUsage(dir)
 	if err != nil {
-		return 0, fmt.Errorf("failed to setup stdout for cmd %v - %v", cmd.Args, err)
+		return 0, err
 	}
-	stderrp, err := cmd.StderrPipe()
+	return usage.Bytes, nil
+}
+
+// Deprecated. Use GetDirUsage instead.
+func GetDirInodeUsage(dir string, timeout time.Duration) (uint64, error) {
```

Comment:
Sorry, you don't need this exact function.  Just `GetDirUsage` is fine.

Reply:
removed its